static/app/components/feedback/feedbackItem/screenshotsModal.tsx

```diff
@@ -34,22 +34,24 @@ export default function ScreenshotsModal({
   return (
     <Fragment>
       <Header closeButton>
-        <PaginationWrapper lightText>
-          <StyledScreenshotPagination
-            nextDisabled={screenshots.length > 1}
-            onNext={() => {
-              setSelectedIndex(prev => prev + 1);
-            }}
-            onPrevious={() => {
-              setSelectedIndex(prev => prev - 1);
-            }}
-            previousDisabled={screenshots.length > 1}
-            headerText={tct('[currentScreenshotIndex] of [totalScreenshotCount]', {
-              currentScreenshotIndex: currentIndex + 1,
-              totalScreenshotCount: screenshots.length,
-            })}
-          />
-        </PaginationWrapper>
+        {screenshots.length > 1 && (
+          <PaginationWrapper lightText>
+            <StyledScreenshotPagination
+              nextDisabled={false}
```

Comment:
@ryan953 the previous logic of `nextDisabled={screenshots.length > 1}` should have been `nextDisabled={screenshots.length <= 1}` actually oops! but since this pagination bar now only shows when there's more than 1 screenshot, this condition doesn't really make sense anymore, so i made the prop always be `false`. not sure if there's a better way to check for disabling the buttons since it looks like the current index can be negative/infinitely increasing

Reply:
if the index is a wild number, positive or negative, then that's ok. keep the buttons enabled!